installer/helm/chart/volcano/config/volcano-admission.conf

```diff
@@ -1,3 +1,4 @@
+#enableHierarchyCapacity: true
```

Comment:
I would like to ask if this new feature needs a feature gate? I have not thoroughly understood this feature, and I am not sure if it will be stable before the next version, so I would tend to discuss whether a feature gate is needed to avoid this problem.